src/ring_theory/roots_of_unity.lean

```diff
@@ -369,6 +369,14 @@ protected lemma not_iff : ¬ is_primitive_root ζ k ↔ order_of ζ ≠ k :=
 ⟨λ h hk, h $ hk ▸ is_primitive_root.order_of ζ,
  λ h hk, h.symm $ hk.unique $ is_primitive_root.order_of ζ⟩
 
+lemma pow_of_div (h : is_primitive_root ζ k) {p : ℕ} (hp : p ≠ 0) (hdiv : p ∣ k) :
```

Comment:
This gives a merge conflict with the other PR, right?
```suggestion
lemma pow_of_dvd (h : is_primitive_root ζ k) {p : ℕ} (hp : p ≠ 0) (hdiv : p ∣ k) :
```

Reply:
You're right. I will fix the conflict once the other PR is merged.